go/private/actions/stdlib.bzl

```diff
@@ -143,6 +143,10 @@ def _build_stdlib(go):
     if not go.mode.pure:
         args.add("-package", "runtime/cgo")
 
+    # For bzltestutil's coverage support.
```

Comment:
We could add these only `if go.coverage_enabled`

Reply:
Good point. Yeah, let's just always build this here and normalize `--collect_code_coverage` in the stdlib transition.